### PR TITLE
Added get function for vmc public IP and corresponding unit tests

### PR DESCRIPTION
**Pull request by navalkp**

### CODE REVIEW

File: src/saltext/vmware/modules/vmc_public_ip.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    )
+    api_url = api_url.format(base_url=api_url_base, org_id=org_id, sddc_id=sddc_id)
+
+    params = vmc_request._filter_kwargs(
+        allowed_kwargs=["cursor", "page_size", "sort_ascending", "sort_by"],
+        cursor=cursor,
+        page_size=page_size,
+        sort_by=sort_by,
+        sort_ascending=sort_ascending,
+    )
```
Comment (waynew) [inline src/saltext/vmware/modules/vmc_public_ip.py:98]: **suggestion**: Params tests are also missing from this PR

**Review by waynew — CHANGES_REQUESTED** [review]

File: src/saltext/vmware/modules/vmc_public_ip.py (hunk 1 above)
Comment (jain-prerna) [reply src/saltext/vmware/modules/vmc_public_ip.py:98]: https://github.com/saltstack/salt-ext-modules-vmware/pull/81#discussion_r721918594

Comment (navalkp) [reply src/saltext/vmware/modules/vmc_public_ip.py:98]: Added params tests in new PR https://github.com/saltstack/salt-ext-modules-vmware/pull/111

Comment (jain-prerna) [reply src/saltext/vmware/modules/vmc_public_ip.py:98]: Current PR has been replaced with new PR #111  along with param test coverage. Hence closing this one.

**Pull request closed without merging**